src/inc/regdisp.h

```diff
@@ -35,6 +35,24 @@ struct REGDISPLAY {
     PCODE   ControlPC;
     TADDR   PCTAddr;
 
+#ifdef WIN64EXCEPTIONS
```

Comment:
You can also ifdef out the pContextForUnwind, it is not used with WIN64EXCEPTIONS. So maybe you can move this block next to the pContextForUnwind so that you can have single #ifdef / #else / #endif there.